canonical-web-and-design/latest-news
# Setup build pipeline

## Done

Setup initial build and testing pipeline for the package

## QA
- Checkout the branch
- Run `npm  run build` and check that a `/dist` directory containing the script has been created
- Run `npm run lint-js` and check that there are no errors
- Run `npm run test` and check that there are no errors


## Issue / Card

Fixes #6
